sky/clouds/service_catalog/os_image/skypilot-gcp-gpu-debian-11.yaml

```diff
@@ -0,0 +1,10 @@
+resources:
+  cloud: gcp
+  accelerators: L4:1
+  use_spot: True
+  disk_size: 50
+  image_id: projects/deeplearning-platform-release/global/images/common-cu121-v20231105-debian-11-py310
```

Comment:
We may want to use the later image, instead of this a bit old one. Let's change it in a future PR then.